server/apiCalls/ladokApi.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const { createApiClient } = require('om-kursen-ladok-client')
+const serverConfig = require('../configuration').server
+
+async function getLadokCourseData(courseCode, testCourse = null) {
```

Comment:
I agree with @allazis! 

I get way you did this way because existing code in `koppsApi.js` has this testCourse param, but I think it would be good to just keep the test/mock data in test-function.

Reply:
Thanks for the great points that you mentioned. I will make the required changes as you mentioned 🙌